pandas/core/reshape/melt.py

```diff
@@ -0,0 +1,101 @@
+# pylint: disable=E1101,E1103
+# pylint: disable=W0703,W0622,W0613,W0201
+from pandas.compat import range, text_type, zip
+from pandas import compat
+from functools import partial
+import itertools
+import re
+
+import numpy as np
+
+from pandas.core.dtypes.common import (
+    _ensure_platform_int,
+    is_list_like, is_bool_dtype,
+    needs_i8_conversion, is_sparse)
+from pandas.core.dtypes.cast import maybe_promote
+from pandas.core.dtypes.missing import notna
+import pandas.core.dtypes.concat as _concat
+
+from pandas.core.series import Series
```

Comment:
just import from pandas as much as you can rather than the implementation (e.g. from pandas import DataFrame)